lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -328,25 +328,10 @@ states:
       f2f:
         targetJourney: F2F_HAND_OFF
         targetState: START
-
-  PYI_CRI_ESCAPE_NO_F2F:
```

Comment:
For P2 we should probably keep the old states initially (and remove in a follow-up PR) to stage this out in case there any in flight journeys in these states as the deploy goes through